Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.t
 # copy bot binary
 COPY --from=builder /build/target/release/turto .
 COPY --from=builder /build/config.toml.template ./config.toml
-COPY --from=builder /build/help.toml.template ./help.toml
-COPY --from=builder /build/templates.toml.template ./templates.toml
+COPY --from=builder /build/help.toml.zh-Hant.template ./help.toml
```

Comment:
These will be revert back to the English version, since Traditional Chinese is actually optional (maybe make a Traditional Chinese package separately?)